gax/src/main/java/com/google/api/gax/tracing/TracedServerStreamingCallable.java

```diff
@@ -59,6 +63,17 @@ public TracedServerStreamingCallable(
     this.tracerFactory = Preconditions.checkNotNull(tracerFactory, "tracerFactory can't be null");
     this.spanName = Preconditions.checkNotNull(spanName, "spanName can't be null");
     this.innerCallable = Preconditions.checkNotNull(innerCallable, "innerCallable can't be null");
+
+    this.tracedFirstCallable =
+        new TracedUnaryCallable<>(
+            innerCallable.first(),
+            tracerFactory,
+            SpanName.of(spanName.getClientName(), spanName.getMethodName() + ".first"));
```

Comment:
Should we follow some strict naming pattern for things like this. If I remember correctly for LRO, `.Start` and `.End` were starting from a capital letter (here it is not the case). Please make naming consistent (or explain why it is not).

Reply:
fixed